js/src/redux/providers/registry/middleware.js

```diff
@@ -91,6 +131,17 @@ export default (api) => (store) => {
           clearTimeout(timeout);
         }
 
+        write.flush();
+
+        break;
+      case 'setReverse':
+        write(
+          () => store.getState().nodeStatus.netChain,
```

Comment:
Premature optimization. 😉  `write` gets called quite often. For all past & new reverse events.